ocs_ci/helpers/dr_helpers_ui.py

```
@@ -902,63 +911,110 @@ def verify_pending_cleanup_alert_firing(
-    except (NoSuchElementException, StaleElementReferenceException) as e:
-        log.warning(
-            f"Could not expand Critical alerts section: {e}. "
-            "Proceeding to verify alert presence directly."
+    if get_semantic_ocp_running_version() >= VERSION_4_22:
+        # On OCP 4.22+ the MCO status-card is gone; alerts are in the OCP
+        # notification drawer (bell icon).  Open it to make the alert elements
+        # visible in the DOM, then check with the drawer-anchored locator
+        # (overridden in acm_configuration_4_22).
+        log.info(
+            "OCP >= 4.22: opening notification drawer to check for "
+            f"'{constants.ALERT_APPLICATION_CLEANUP_PENDING}' alert"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)